### PR TITLE
added Dutch language for interface

### LINKED ISSUE #261: Dutch translation
### Prerequisites

- [x] I'm using latest version: v 

### Description

translating Stretchly for Dutch language

Operating system: MacOS High Sierra version 10.13.3

### PR DESCRIPTION
**Pull request by Sveijk**
Issue: close #261 

### Requirements

- [x]  issue was opened to discuss proposed changes before starting implementation.
- [x]  during development, `node` version specified in `package.json` was used.
- [x]  package versions were not changed.
- [x]  app version number was not changed.
- [x]  all new code has tests to ensure against regressions.
- [x] `npm run lint` reports no offenses.
- [x] `npm run test` is error-free.
- [x]  README and CHANGELOG were updated accordingly.

### Description of the Change

Added a Dutch Language translation @aleuca & @Sveijk

### CODE REVIEW

File: .idea/workspace.xml [hunk 1]
```diff
@@ -0,0 +1,350 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (hovancik) [inline .idea/workspace.xml]: Please add this to `.gitignore`

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
```
Comment (hovancik) [inline CHANGELOG.md]: You do not need to add `## [Unreleased]` again ;] Just put it under the already existing one. 

More about format here: https://keepachangelog.com/en/1.0.0/

File: package-lock.json [hunk 3]
```diff
@@ -10,9 +10,9 @@
       "integrity": "sha512-juYJNi8JEpTUWXwz8ssa8Oop4n/kwJ/pIQP22vJAVAe6RTRD+0m+e9LRNnfK2EDaX8uwmUzLNGviFQRD6SxeOw==",
       "dev": true,
```
Comment (hovancik) [inline package-lock.json]: Not sure why this file was changed, it should not be changed.

File: .idea/workspace.xml (hunk 1 above)
Comment (Sveijk) [reply .idea/workspace.xml]: I will

Comment (Sveijk) [reply .idea/workspace.xml]: Ohw, this is what @aleuca has been working on.. we'll check this together

**Comment by aleuca** [comment]
Changed the versions back and updated CHANGELOG.md

**Comment by aleuca** [comment]
Added workspace file to gitignore

**Pull request merged**